packages/client/wallets/aa/src/adapters/reservoir.ts

```diff
@@ -4,19 +4,20 @@ import { hexToBigInt, http } from "viem";
 import { getBundlerRPC } from "../blockchain/BlockchainNetworks";
 import { EVMAAWallet } from "../blockchain/wallets/EVMAAWallet";
 
+// TODO does this still all work?
```

Comment:
Yes, you can test it [here](https://github.com/Crossmint/scw_demo/blob/main-w3a/src/app/utils/reservoir.ts)